docs/index.md

```diff
@@ -20,6 +20,6 @@ Our smart contracts are hosted at [Contracts](https://github.com/razor-network/c
 Please note that the project is in active development and breaking changes are introduced frequently.
 
 ##Client
-A javascript reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/cli)
+A javascript reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/razor-go)
```

Comment:
It is a GoLang based CLI now. Not javascript.
Also, note that `##` is used carelessly in this file. Please fix those as well.